src/screens/WalletConnect/CreateTokenScreen.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+} from 'react-native';
+import { t } from 'ttag';
+import HathorHeader from '../../components/HathorHeader';
+import OfflineBar from '../../components/OfflineBar';
+import { CreateTokenRequest } from '../../components/WalletConnect/CreateTokenRequest';
+import { COLORS } from '../../styles/themes';
+
+export function CreateTokenScreen({ route }) {
+  const { createTokenRequest } = route.params;
+
+  return (
+    <Wrapper>
+      <HathorHeader
+        title={t`Create Token Request`.toUpperCase()}
```

Comment:
Is it missing the translation files in this PR?

Reply:
Yes it was, fixed